### DISCUSSION TITLE
Style Editor panels remember selected tab.

### DISCUSSION THREAD

**Discussion by gab3d** [opentoonz/opentoonz#6735] [Ideas]
Along the lines of testing some slight UI improvements that could allow for better utilization of the current features of the interface.
I'd like to attract some attention to this one:

If each Style Editor panel could remember its selected tab when the program is restarted, we could use a Room like this one, to more comfortably paint with MyPaint brushes:
![mypaint_room](https://user-images.githubusercontent.com/18104091/65380349-28e59480-dcb0-11e9-894f-a540e4fdeff5.png)

Unfortunately, when we restart OpenToonz this room looks like this:
![mypaint_room (reopened)](https://user-images.githubusercontent.com/18104091/65380318-5a119500-dcaf-11e9-8298-0d8f50c67d14.png)

if each Style Editor panel could just remember which tab was selected, a room like this could be a great starting point for painting with MyPaint brushes.

**Comment by artisteacher** [comment]
I completely agree!

**Comment by RodneyBaker** [comment]
_ Hidden to keep the report on topic_

> slight UI improvements

I look forward to exploring your deeper UI improvements!  :)

Aside:  I would say workflow with Palettes is one of my main (although minor) frustrations.
I constantly have to reopen things like the Palette window... move them around... reopen again... move again...
That's mostly on me of course.  If I would take the time to learn the shortcut keys I'd probably be fine.

(Don't get me in trouble!  I created a Demo / Research tag as a way to keep feature requests open on the sly for all those active here but I don't see that ending well.  Just joking... mostly!) ;)
